docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -165,7 +170,7 @@ It is not recommended to run scalable mode with `filesystem` storage. For the pu
 
 ## Object Storage Configuration
 
-After testing Loki with MinIO, it is recommended to configure Loki with an object storage provider. The following examples shows how to configure Loki with different object storage providers:
+After testing Loki with MinIO, we recommend to configure Loki with an object storage provider. The following examples shows how to configure Loki with different object storage providers:
 
 {{< admonition type="caution" >}}
 When deploying Loki using S3 Storage **DO NOT** use the default bucket names;  `chunk`, `ruler` and `admin`. Choose a unique name for each bucket. For more information see the following [security update](https://grafana.com/blog/2024/06/27/grafana-security-update-grafana-loki-and-unintended-data-write-attempts-to-amazon-s3-buckets/). This caution does not apply when you are using MinIO. When using MinIO we recommend using the default bucket names.
```

Comment:
outside the scope of this PR so ignore me but we probably shouldn't ever have this in the docs. The helm chart should ideally block you from foot-gunning in this particular way

Reply:
Correct it is less a problem now since AWS has changed their billing so users hopefully won't get changed crazy amounts of money. I belive we don't set these anymore but would be good to see if we can check to make sure someone aren't using these bucket names unless using minIO